CONTRIBUTION.md

```diff
@@ -0,0 +1,60 @@
+# Contributing Guidelines:
+
+We welcome all contributions to the Hydrator project. The contribution can be a [issue report](https://github.com/devcer/Hydrator/issues) 
+or a [pull request](https://github.com/devcer/Hydrator/pulls).
+
+## Contribution workflow:
+
+
+1. Check that there isn't [already an issue](https://github.com/devcer/Hydrator/issues) about your idea to avoid duplicating work.
+    * If there isn't one already, please create one so that others know you're working on this
+
```

Comment:
@vismitap Can you also add star the repo to the workflow? May be an additional step below step 1?

Reply:
Sure! Please give me some time, working on that!

Reply:
@devcer I searched about this and could not find any method to link the star the repo thing to the readme file.  The maximum we can do is link the /stargazers to view the people who have starred the repo or link it to the original page (https://github.com/devcer/Hydrator). There's no direct way to star the repo from the readme itself, the user will have to go to the original repo page and star it from there. Should I go ahead with adding the original repo 
link?

Reply:
@vismitap you can do exactly what you did with fork the repo step. Maybe I wasn't clear before, sorry bout it.
`Star the [devcer/Hydrator repository](https://github.com/devcer/Hydrator) to your GitHub account.`
Add emoticons for star, fork, and clone if possible.

Reply:
Oh okay!